CONTRIBUTING.md

```diff
@@ -1,18 +1,17 @@
-﻿# Filing an issue
+# Filing an issue
```

Comment:
Yes, there was something, must have been that, but I did all edits online, so can not be sure.